src/pages/WriteIntroductionPage.tsx

```diff
@@ -1,9 +1,83 @@
 import type React from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type FormData = {
+	name: string;
+	introduction: string;
+	githubUrl: string;
+	XUrl: string;
```

Comment:
```suggestion
	githubUrl?: string;
	XUrl?: string;
```
オプショナルらしいので、こんな感じで。? がないと、undefined を許さないという意味になります！